pkg/detectors/jiratoken/jiratoken.go

```diff
@@ -73,9 +73,9 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 					Raw:          []byte(resToken),
 					RawV2:        []byte(fmt.Sprintf("%s:%s:%s", resEmail, resToken, resDomain)),
 				}
-				s1.ExtraData = map[string]string{
-					"rotation_guide": "https://howtorotate.com/docs/tutorials/atlassian/",
-				}
+				//s1.ExtraData = map[string]string{
```

Comment:
@ankushgoel27 any reason why this is commented out?

Reply:
i initially commented them out due to go test command failing for not ignoring extradata in the test file. I added that and now the test's pass.

i believe this is ready for merge. Both detectors work and both tests pass.